docs/source/index.rst

```diff
@@ -107,8 +110,9 @@ The script essentially consists of four steps:
     The suffix denotes the quantum engine that underlies the implementation.
  2. Define the required ``structure`` and ``engines`` inputs.
  3. Retrieve the workflow builder instance for the given inputs.
-    This ``get_builder`` method will return a `process builder instance`_ that has all the necessary inputs defined based on the protocol of the input generator.
-    At this point, you are free to change any of these default inputs.
+    This ``get_builder`` method will return a `process builder instance`_ that has all the necessary inputs defined
+    based on the protocol (see next section) of the input generator.
+    At this point, a user is free to change any of these default inputs.
```

Comment:
Same here, think the only change is that the sentence was split into two lines.
For git diff purposes, it is better to keep every sentence on a single line, so let's keep that convention.
You can use soft-wrapping in your editor if you prefer to have a shorter line length